package.json

```diff
@@ -4,23 +4,23 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "new": "ts-node src/new-intent.ts",
-    "replay": "ts-node src/existing-intent.ts",
-    "address": "ts-node src/get-address.ts"
+    "new": "tsx src/new-intent.ts",
+    "replay": "tsx src/existing-intent.ts",
+    "address": "tsx src/get-address.ts"
```

Comment:
Theoretically, it's possible to make `ts-node` run ESM TS files with some esoteric config, but `tsx` provides that out of the box.

Reply:
ok cool dont think it matters whether we use ts-node or tsx